ui/tests/e2e/indexui.spec.js

```diff
@@ -22,4 +22,24 @@ test.describe('Index Page UI Tests', () => {
   test('Test if Profile button is displayed', async ({ page }) => {
     await expect(page.locator('[data-test=profile-button]')).toBeVisible();
   });
+
+  test('Test if Dashboard is displayed', async ({ page }) => {
+    await expect(page.locator('[data-cy=dashboard]')).toBeVisible();
+  });
+
+  test('Test if Lifecycle is displayed', async ({ page }) => {
+    await expect(page.locator('[data-cy=lifecycle]')).toBeVisible();
+  });
+
+  test('Test if Configuration is displayed', async ({ page }) => {
+    await expect(page.locator('[data-cy=configuration]')).toBeVisible();
+  });
+
+  test('Test if Performance is displayed', async ({ page }) => {
+    await expect(page.locator('[data-cy=performance]')).toBeVisible();
+  });
+
+  test('Test if Extensions is displayed', async ({ page }) => {
+    await expect(page.locator('[data-cy=Extensions]')).toBeVisible();
+  });
```

Comment:
The tests are failing as it exceeds the 1 min time limit to find the element, I tried by editing for dashboard here is the results
![image](https://github.com/user-attachments/assets/2d56f2fb-9fe2-4c96-b6ba-cffa203b8761)

Reply:
@ianrwhitney i think that's why the code that was present was using `locator ` and `data-cy=data-testid`  because they were able to verify the elements under 60 seconds.

Reply:
@THEBOSS0369 We don't want to add like this, we want a test to testing our existing functionality is work, this type of test just will cost as more in term of maintaining them

https://kentcdodds.com/blog/the-testing-trophy-and-testing-classifications

Reply:
@jerensl ! I didn't get which are u referring to please confirm me so that i can proceed further.
1. This is the code that is present in my PR and passes all the tests
 
```javascript
  test('Test if Dashboard is displayed', async ({ page }) => {
    await expect(page.locator('[data-cy=dashboard]')).toBeVisible();
  });
``` 
2. This is the code that was suggested by @ianrwhitney , but tests fails for this one 
```javascript
  test('Test if Dashboard is displayed', async ({ page }) => {
    await expect(page.getByTestId('dashboard')).toBeVisible();
  });
``` 
Out of these 2 codes whom were u referring please clarify so that i can clear my confusion.